### ISSUE TITLE
A11y_AzureDataStudio_BigDataClusterOverviewPage _ScreenReader : NVDA is not announcing status message for the "Copy" button.

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7660]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."
​
GitHubTags:#A11y_AzureDataStudio;#A11yWCAG2.1;#A11yTCS;#WCAG4.1.3;#DesktopApp;#NVDA;#JAWS;#SQL Azure Data Studio;#Win10

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362
NVDA Version: 2019.2
JAWS Version:2019.1907.42 ILM

**Additional Details:**
Tool Name : NVDA , JAWS

**Steps to Reproduce:**
1. Install and Launch the application the azure data studio application.
2. Open the “Connections” viewlet (CTRL + SHIFT + D)
3. Navigate to the “SQL Server Big Data Clusters” section and expand it
4. Navigate to the "Add SQL Server big data cluster controller…" node and click on it.
5. Fill in the connection information for the BDC Controller Endpoint
6. Click “Add”
7. Navigate to the controller node in the “SQL Server Big Data Clusters” view and right click on it, then select “Manage” to open the dashboard
8. "Big Data Cluster Overview page " should open.
9. Start NVDA.
10. Navigate till "Copy" button and click on it .
11. Observe the announcement by screen reader .

**Actual:**
After clicking on "Copy" button data get copied but NVDA is not announcing status message for the "Copy" button.
And other issue is that it does not announce what kind of the data will get copied.
**Similar issue is also observed with JAWS .**
 
**Expected:**
NVDA should announce status message for the "Copy" button.
for example ,"End Point link" has been copied .

**User Impact:**
Screen reader user will find it difficult to understand that data has been copied or not .

**Attachment for Reference:**
[7660_A11y_AzureDataStudio_BigDataClusterOverviewPage _ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3716232/7660_A11y_AzureDataStudio_BigDataClusterOverviewPage._ScreenReader.pptx)

**Comment by 12-shweta** [comment]
Verified the bug in below environment :
Version: 1.15.0-insider (user setup)
Commit: 041f34b
Date: 2020-01-08T05:36:19.161Z
VS Code: 1.38.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18363
Issue is fixed and does not reproduce .
![7660_Closed](https://user-images.githubusercontent.com/54656883/72725533-01460080-3bac-11ea-8a87-5ec41404ca1a.png)
